lib/pbench/server/auth/auth.py

```diff
@@ -54,7 +54,11 @@ def encode_auth_token(self, time_delta: datetime.timedelta, user_id: int) -> str
 
     def get_secret_key(self):
         try:
-            return os.getenv("SECRET_KEY", "my_precious")
+            return (
+                OpenIDClient.PUBLIC_KEY
+                if OpenIDClient.PUBLIC_KEY
+                else os.getenv("SECRET_KEY", "my_precious")
+            )
```

Comment:
I'm sad to say it, but this is not a good place for a ternary.  Instead,
```
            if OpenIDClient.PUBLIC_KEY:
                return OpenIDClient.PUBLIC_KEY
            return os.getenv("SECRET_KEY", "my_precious")
```
is just as clear (if not actually clearer...), and it's two lines shorter.

Reply:
Decided not to update this function at this time https://github.com/distributed-system-analysis/pbench/pull/3095#discussion_r1038495643